package.json

```diff
@@ -1,58 +1,60 @@
-{
```

Comment:
Why did the entire file get rewritten?  It would be nice to keep the history for all the different lines in this file.

Reply:
I guess it's tabs vs spaces, I can reformat

Reply:
yeah, sounds good!